### ISSUE TITLE
[BUG] how to use pdf dark mode without using the UI dark mode

### ISSUE THREAD

**Issue by Darcake** [tefkah/zotero-night#147]
**Describe the bug**
Hello, I'd like to consult a question:
The pdf can not be set to dark mode when the UI is in day mode? How can i use pdf dark mode only.

**OS (MacOS XX.xx [specify Intel or Silicon], Windows, Linux [specify distro/package type]:**
win11

**Zotero Night version (please check that you are on the latest version)**
0.4.23

**Comment by tefkah** [comment]
Currently this is not possible, and I don't think I will support this for the Zotero 6 version.

Reason being that I am kinda hesitant to do anything that has an effect on "normal" zotero. It has happened before that my plugin ended up affecting the normal functioning of the ui, which people then go to the Zotero devs for, which causes them to spend a bunch of time debugging only to find out it was my fault. Then they get mad at me (understandably), which could lead to them banning this extension, which we probably don't want.

TL;DR Probably not, but i do plan on supporting more themes in the future, which may be lighter.
